circle.yml

```diff
@@ -11,8 +11,11 @@ dependencies:
   # just to pull each time. #rollondockercaching
   override:
     - docker pull condaforge/linux-anvil
+    - ./ci_support/run_docker_build.sh || touch ./build_failed
```

Comment:
CircleCI cache happens after the dependencies phase, unlike Travis' which happens at the end of the whole build.
So I had to move the build command and have a way to detect if it failed later.